nc-times/src/components/AllArticles.jsx

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from "react"
+import { getAllArticles } from "../api"
+import GoodBox from "../resources/GoodBox"
+
+function AllArticles(){
+
+const [articles , setArticles] = useState([])
+useEffect(()=>{
+    getAllArticles()
+    .then((data)=>{
+       
+        setArticles(data)
+        
+    })
+}, [])
+    
+
```

Comment:
You need some loading state in here to feed back to the user while they're waiting for the page to render.